helper/schema/schema_test.go

```diff
@@ -111,7 +111,7 @@ func TestValueType_Zero(t *testing.T) {
 		{TypeString, ""},
 		{TypeList, []interface{}{}},
 		{TypeMap, map[string]interface{}{}},
-		{TypeSet, nil},
+		{TypeSet, new(Set)},
```

Comment:
This seems to be the correct behavior. The zero value of a set is an empty set, not a nil value.